### DISCUSSION TITLE
General availability of `Avalonia.Automation`

### DISCUSSION THREAD

**Discussion by ChristoWolf** [AvaloniaUI/Avalonia#8254] [Q&A] [answered]
Hi!

Seeing that `Avalonia.Automation` (added with #5177) seems to not yet be generally available in the NuGet release, I wanted to ask what the roadmap/ETA for this feature is?
I would have strong need for these automation features, as them missing is the only reason why we cannot use Avalonia at the moment.
Otherwise it would be absolutely perfect for my needs :)

**Comment by hez2010** [comment]
The implementation (especially Win32) of `Avalonia.Automation` needs migrate to COMWrappers before it can be general available, otherwise it won't support trimming at all.

**Comment by maxkatz6** [comment, marked as answer]
First preview will be available in the end of June or beginning of the July.
Meanwhile you can use [nightly builds](https://github.com/AvaloniaUI/Avalonia/wiki/Using-nightly-build-feed) for testing, but it's not compatible with some third-party libraries and has [breaking changes](https://github.com/AvaloniaUI/Avalonia/wiki/Breaking-Changes).

**Comment by ChristoWolf** [comment]
Thanks for all the helpful replies, I am excited to try this out soon!
